en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -9,9 +9,6 @@ It is suitable for testing multi-vehicle support in *QGroundControl* (or the [MA
 [Multi-Vehicle Simulation with Gazebo](../simulation/multi-vehicle-simulation.md) should be used for swarm simulations with many vehicles, or for testing features like computer vision that are only supported by Gazebo.
 :::
 
-:::note
```

Comment:
FYI removed note. It is correct, but PX4 v1.8 is no longer relevant, so this does not need to be said.